### PR TITLE
US247Task249 API call return sprint velocity (unit points) based on sprint Id

### PR DESCRIPTION
**Pull request by xiangweizheng**
US247Task249 Create a call return sprint velocity (unit points) based on sprint Id

Test Code:
import * as taiga from '../libraries/Taiga';
 {console.log(taiga.sprint_velocity_pts(205316).then((val) => {console.log('fe', val)} ))}

Expected result:
fe 46.5
It indicate sprint velocity is 46.5 for this sprint.

### CODE REVIEW

File: taigit/src/libraries/Taiga.ts [hunk 1]
```diff
@@ -498,4 +498,29 @@ taiga_issues(projId : number) : Promise<Object>{
...
+    let data = await axios.get("https://api.taiga.io/api/v1/milestones/"+sprintId.toString()+ '/stats');
+    //test id:  https://api.taiga.io/api/v1/milestones/205316/stats
+    //https://api.taiga.io/api/v1/milestones/219984/stats
+
+     let entry = {
+            pts : data.data.completed_points
+        }
+    let velocity : number = 0;
+    for(let each of entry.pts ){
+        velocity += each;
```
Comment (trevorforrey) [inline taigit/src/libraries/Taiga.ts]: Does this just add up the total points? Shouldn't the velocity be `total number of points completed / total points` ?

Comment (xiangweizheng) [reply taigit/src/libraries/Taiga.ts]: For a single sprint velocity, it is the completed points.

File: taigit/src/libraries/Taiga.ts [hunk 2]
```diff
@@ -498,4 +498,29 @@ taiga_issues(projId : number) : Promise<Object>{
...
+/**
+ * @summary This call return sprint velocity (unit points) based on sprint Id
+ * @param sprintId the ID for the access of sprint stats
+ * @returns
+ * * {
+ *          velocity : number //sprint velocity
+ * }
+ */
+export async function
+sprint_velocity_pts(sprintId : number) : Promise<Object> {
```
Comment (zawata) [inline taigit/src/libraries/Taiga.ts]: @returns should be a number not an object

return type should be Promise<number>

File: taigit/src/libraries/Taiga.ts [hunk 3]
```diff
@@ -498,4 +498,29 @@ taiga_issues(projId : number) : Promise<Object>{
...
+    let data = await axios.get("https://api.taiga.io/api/v1/milestones/"+sprintId.toString()+ '/stats');
```
Comment (zawata) [inline taigit/src/libraries/Taiga.ts]: you should be using existing functions not remaking them. ie: `sprint_stats()`

Comment (xiangweizheng) [reply taigit/src/libraries/Taiga.ts]: Run into error use sprint_stats(). 
First of all, I need to make this API working so I use this way.

File: taigit/src/libraries/Taiga.ts [hunk 4]
```diff
@@ -498,4 +498,40 @@ taiga_issues(projId : number) : Promise<Object>{
...
+/**
+ * @summary This call return sprint velocity (unit points) based on sprint Id
+ * @param sprintId the ID for the access of sprint stats
+ * @returns
+ * * {
+ *          sprint_end : boolen // if sprint end currently
+ *          velocity : number //sprint velocity
+ * }
+ */
+
```
Comment (HalcyonAura) [inline taigit/src/libraries/Taiga.ts]: Remove all these extra blank lines

**Pull request merged**